circom/aes256.go

```diff
@@ -0,0 +1,318 @@
+package circom
+
+import (
+	"github.com/consensys/gnark/frontend"
+)
+
+type AES256Wrapper struct {
+	Plain  [16]frontend.Variable
+	Key    [32]frontend.Variable
+	Cipher [16]frontend.Variable `gnark:",public"`
+}
+
```

Comment:
```suggestion

var _ = frontend.Circuit(&AES256Wrapper{})

````
Just to be more formal, not a functional change.